card-unification.md

```diff
@@ -0,0 +1,90 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `buffer`, `isInEditor`, `options`, `editorEnv` and `payload` properties
```

Comment:
I considered `isEditing` but to me it implies that the card is in "edit mode" (not "display mode"). `isInEditor` is intended to capture the fact that the card is being rendered for an editor (whether it is in edit or display mode at the moment). Same thing for `editorEnv` — that is an object of properties that could be useful when a card is being displayed/edited in an editor. `editorEnv.save` seems like it would be useful any time for a card in an editor (`save(newPayload, false)` is the way to update a card's payload without transitioning a card). `editorEnv.edit` is a no-op when the card is already in edit mode. It's redundant but doesn't seem confusing to me — I wouldn't expect someone to think that calling "edit" again while in edit mode would do something.